util/dvsim/SimResults.py

```diff
@@ -96,6 +96,21 @@ def _add_item(self, item, results):
 
     def _add_run(self, item, status):
         '''Add an entry to table for item'''
+
+        def _find_max_time(a, b):
+            '''Helper function to find the max time.
+               Currently only support the following input formats:
+               1). ""
+               2). "XXX.XX"
+               3). "HR:MIN:SEC"
+            '''
+
+            if "." in a and "." in b:
+                return a if float(a) > float(b) else b
+            else:
+                # Can directly compare the two strings if they are not floats.
+                return a if a > b else b
+
```

Comment:
After parsing the CPU / sim time from the log, we should maintain in integer format, representing time in seconds / microseconds. So this max computation is simpler.